content/blog/2020/10/Camel36-Whatsnew/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Apache Camel 3.6 What's New"
+date: 2020-10-21
```

Comment:
Won't be published unless:
```suggestion
date: 2020-10-21
draft: false
```

Reply:
Its a copy from 3.5 blog post that was published